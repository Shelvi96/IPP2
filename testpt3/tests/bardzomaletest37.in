NEW ID

8139217:5>0223
5;473>0595700:7
68072221;:>64849;
8>898967
7594394>0:552
:8;:00:891>:154073
8;>682:67702
817865>7
4532139>0:844476
6139>025
83935>0972
939>;53409;722
06:97;046>1;7953684
7:83>921
593;>0:3121;
0:55531:0>0042
;8045379>6080
;>6950705
0690>31:316
@5:03;08508;12;865973:;22145
@26385738;;1;84124181;3497
@38716:;1:8014019;1
@43:
@;317722::26
@882263
@;76096532
@1484533092:;01;831668
@62:703:70;;0
@97
@940774860;06:848:23391:
@491;609742236:2;2294
@1047375856070;5
