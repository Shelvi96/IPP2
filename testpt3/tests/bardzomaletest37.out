1304801430617088
1010020010000
100103
0
0
0
0
4043699
0
0
26230231412
10130110
81
